unity-ggjj/Assets/Tests/EditModeTests/NarrativeScriptTests.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Generic;
+using Ink;
+using Moq;
+using NUnit.Framework;
+using UnityEngine;
+
+public class NarrativeScriptTests
+{
+    public readonly string TestScript = "This is a test script" +
```

Comment:
Nit: As it's not an interpolated string, this field can actually be marked as a constant:
```suggestion
    public const string TestScript =    "This is a test script" +
```